### ISSUE TITLE
Too Many Tables error

### ISSUE THREAD

**Issue by earney** [gluesql/gluesql#570]
```

gluesql> create table mytable (id int(8), name text);
gluesql> create table mytable1 (id int(8), name text);
gluesql> select a.* from mytable a, mytable b where a.id=b.id;
[error] unimplemented - select on two or more than tables are not supported
```
The select statement is valid SQL.  The error occurs in the translate_select which is located in the translate_query.rs file.
One question..  Should we create a new file called something like translate_select.rs that will take SqlSelect struct and produce something we can work with, or should we just expand the translate_select function?   I am thinking the general solution to this problem maybe a bit wordy, so creating a new file, etc maybe the way to do go, but I'm wanting feedback before I get started.  Any comments, ideas?

**Comment by panarch** [comment]
There is a good keyword for this issue - supporting implicit inner join syntax

ref. https://stackoverflow.com/questions/44917/explicit-vs-implicit-sql-joins

```sql
select a.* from mytable a, mytable b where a.id=b.id;
```
Currently GlueSQL only supports explicit inner join
```sql
SELECT a.* FROM mytable a JOIN mytable b WHERE a.id = b.id;
```

btw. it would be better to have different error name.
```
[error] unimplemented - select on two or more than tables are not supported
```
to
```
[error] unimplemented - implicit inner join is not supported
```
